New version: AStarSearch v0.7.0

JuliaRegistrator opened the pull request:
- Registering package: AStarSearch
- Repository: https://github.com/PaoloSarti/AStarSearch.jl
- Created by: @PaoloSarti
- Version: v0.7.0
- Commit: 221fc331ab24912fb269211eb76a1541e5e2519b
- Git reference: HEAD
- Description: A* Algorithm in Julia
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
Breaking Change:
- Requires at least julia 1.6
`````
<!-- END RELEASE NOTES -->

PaoloSarti commented:
There are actually no breaking changes, can be closed, I will open a patch release

JuliaHubRegistrator committed:
New version: AStarSearch v0.7.0

UUID: e6cbe913-2b79-4cc5-848a-e3bbf8537828
Repo: https://github.com/PaoloSarti/AStarSearch.jl.git
Tree: 3bf7e54baeab28992daf4f5c7b2c7dbb88df28e9

Registrator tree SHA: be588f25c435fa5b456b986adf2f5c02bd3298f2
